client/src/views/HomeView.vue

```diff
@@ -1,108 +1,72 @@
 <template>
   <div class="home">
     <!-- Login Button -->
-    <v-btn
-      elevation="2"
-      @click="$router.push('/login')"
-    >
-      LOGIN
-    </v-btn>
-    <br><br><br>
+    <v-btn elevation="2" @click="$router.push('/login')"> LOGIN </v-btn>
```

Comment:
When using the Vue Router, a best practice is to pass in a routing object as opposed to a string.  [Vue Routing Docs](https://router.vuejs.org/guide/essentials/navigation.html#navigate-to-a-different-location)

This could be rewritten as $router.push({ name: 'login' })